### PR TITLE
LPS-147010 Add condition when the objectViewColumns is empty and objectView is default

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://issues.liferay.com/browse/LPS-147010

### CODE REVIEW

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/index.tsx [hunk 1]
```diff
@@ -129,41 +129,52 @@ const CustomView: React.FC<React.HTMLAttributes<HTMLElement>> = () => {
...
-				headers: HEADERS,
-				method: 'PUT',
-			}
-		);
-
-		if (response.status === 401) {
-			window.location.reload();
-		}
-		else if (response.ok) {
+		if (objectView.defaultObjectView && objectViewColumns.length === 0) {
```
Comment (interaminense) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/index.tsx:134]: Hi @danvb112, could you invert the condition? This way the request is made first and `else` should fire alert

**Review by interaminense — CHANGES_REQUESTED** [review]
Hi @danvb112 let me know if you agree with me.

**Comment by marcelabc** [comment]
hey @danvb112, I put this label because didi asked you a code change :)

**Comment by danvb112** [comment]
Ok @marcelabc, Thanks! :)

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/index.tsx (hunk 1 above)
Comment (danvb112) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/index.tsx]: I agree with you @interaminense , I hadn't thought of that when I made the solution, Thanks! :)

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by danvb112** [comment]
ci:forward

**Pull request closed without merging**